src/components/textfieldControl/css/TextfieldStepper.css

```diff
@@ -23,13 +23,20 @@
 /* Textfield Styling */
 
 .fl-textfieldStepper .fl-textfieldStepper-textField {
+    -moz-appearance: textfield;
```

Comment:
Which current browsers does this tweak display in? Is there a route to eliminating it someday?

Reply:
As the `-moz` prefix would suggest, it's needed for Firefox to display this input without the default stepper controls. The route to eliminating it would be for Firefox to provide an alternative method of doing this that doesn't require a browser prefix.